src/Workspaces/Core/MSBuild/MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -35,6 +35,12 @@ internal sealed class ProjectFileInfo
         /// </summary>
         public string OutputRefFilePath { get; }
 
+        /// <summary>
+        /// The default namespace of the project ("" if not defined, which means global namespace),
+        /// or null if it is unknown or not applicable. 
+        /// </summary>
```

Comment:
What about F#? (or whatever else using Roslyn workspace)

Reply:
good question :)   how does this work for F#? :)  

For now though, i think it would be ok to just have this be in the remarks section.  Namely that is is normally set for C# (possibly to empty), and it is *never* set for VB.  I think it's ok to not mention F# for now...